docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -154,6 +154,28 @@ On CentOS, the correct path is `/usr/pgsql-11/bin/pg_config`.
 
 ## Using PostgreSQL extensions
 
+### file_fdw example
+
+First, install the extension:
+
+```sql
+CREATE EXTENSION file_fdw;
+```
+
+Create a foreign server:
```

Comment:
Do you need to create the wrapper before you create the server? That's how you show it in the Explore page...

Reply:
The `CREATE EXTENSION` command creates the wrapper